### DISCUSSION TITLE
Doubts and Acknowledgments

### DISCUSSION THREAD

**Discussion by GustavoSwDaniel** [miguelgrinberg/Flask-SocketIO#1624] [General]
irst I want to apologize for not having responded to the Issue you helped me, it was a busy month.
Yes, I managed to make flask-socketIO work as a websocket.

So my doubt today is about the logs, when no data is sent

![Captura de tela de 2021-07-05 22-41-49](https://user-images.githubusercontent.com/43218278/124530412-d7ee5400-dde2-11eb-9d49-37f0fae0529b.png)

I would like to know what these logs are, these logs are being generated infinitely

One more question, can flask-socketIO be used for a real-time dashboard?

**Comment by miguelgrinberg** [comment]
The ping/pong packets are internal to the Socket.IO protocol. You should see a pair of these every 25 seconds for each connected client.

> One more question, can flask-socketIO be used for a real-time dashboard?

I guess? This package only cares about sending and receiving messages, what you do with that is totally up to you.
